### PR TITLE
issue #48 fix: dialog windows

### PR DESCRIPTION
**Pull request by joaaoleite**

### CODE REVIEW

File: assets/crx/js/embed.js [hunk 1]
```diff
@@ -5,6 +5,13 @@ var webview = document.getElementById('webview');
 var findBox = document.getElementById('find-box');
 var findInput = document.getElementById('find-text');
 
+// Get dialog box
+var dialogBox = document.getElementById("dialog-box");
+var dialogText = dialogBox.getElementsByTagName('div')[0];
```
Comment (eladnava) [inline assets/crx/js/embed.js]: Maybe we can use `dialogBox.querySelector();` and also for the `input` and two `button` elements?
https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector

File: assets/crx/js/embed.js [hunk 2]
```diff
@@ -82,4 +89,39 @@ function copyToClipboard(str, mimetype) {
 
     // Execute browser command 'Copy'
     document.execCommand("Copy", false, null);
-}
\ No newline at end of file
+}
+
+// Dialogs custom box
+var DialogController = null;
+dialogBox.getElementsByTagName('button')[0].addEventListener('click',function(){
```
Comment (eladnava) [inline assets/crx/js/embed.js]: Please make use of the `dialogCancel` variable here.

File: assets/crx/js/embed.js [hunk 3]
```diff
@@ -82,4 +89,39 @@ function copyToClipboard(str, mimetype) {
...
+    DialogController.cancel();
+    dialogBox.style.display = 'none';
+});
+dialogBox.getElementsByTagName('button')[1].addEventListener('click',function(){
```
Comment (eladnava) [inline assets/crx/js/embed.js]: Please make use of the `dialogOK` variable here.

File: assets/crx/js/embed.js [hunk 4]
```diff
@@ -82,4 +89,39 @@ function copyToClipboard(str, mimetype) {
 
     // Execute browser command 'Copy'
     document.execCommand("Copy", false, null);
-}
\ No newline at end of file
+}
+
+// Dialogs custom box
+var DialogController = null;
```
Comment (eladnava) [inline assets/crx/js/embed.js]: Please camel-case this variable (`dialogController`) to conform with the project coding style. 👍

File: assets/crx/html/embed.html [hunk 5]
```diff
@@ -15,6 +15,14 @@
         <input type="text" id="find-text" placeholder="Find in page" />
     </div>
 
+    <!-- Alert/Confirm box -->
+    <div id="dialog-box">
+      <div>Example dialog text</div>
```
Comment (eladnava) [inline assets/crx/html/embed.html]: Please add `id=` or `class=` to the `div`, `input`, and two `button` elements, as referring to them via their index or tag name is risky and may break in the future if we introduce other elements here.

**Comment by joaaoleite** [comment]
@eladnava, I reviewed the code already!  👍

**Comment by eladnava** [comment]
@joaaoleite Thanks so much for the PR! I have made some slight changes and deployed it to applicationize.me. 👍

**Pull request merged**